lib/common/xml_io.c

```diff
@@ -24,46 +24,6 @@
 #include <crm/common/xml_io.h>
 #include "crmcommon_private.h"
 
-/* @COMPAT XML_PARSE_RECOVER allows some XML errors to be silently worked around
```

Comment:
I thought I remembered that *for some reason* we had decided to wait on this beyond 3.0.0... But I have no idea why that might have been, and as of https://github.com/ClusterLabs/pacemaker/pull/3361#discussion_r1490096446 we were still talking about dropping it. So I'm happy to assume I'm misremembering.

Reply:
We hadn't expected that something like `cibadmin -C -o nodes --xml-text '<node id="node1" uname="node1">'` (missing close at end of tag) would stop working. But pcs and crm shell do not appear to do anything like that, so we're better off moving forward.